paymentsheet/src/main/java/com/stripe/android/paymentsheet/analytics/PaymentSheetEvent.kt

```diff
@@ -39,18 +39,29 @@ internal sealed class PaymentSheetEvent : AnalyticsEvent {
         initializationMode: PaymentSheet.InitializationMode,
         orderedLpms: List<String>,
         duration: Duration?,
+        linkMode: LinkMode?,
         override val isDeferred: Boolean,
-        override val linkEnabled: Boolean,
         override val googlePaySupported: Boolean,
     ) : PaymentSheetEvent() {
         override val eventName: String = "mc_load_succeeded"
+        override val linkEnabled: Boolean = linkMode != null
         override val additionalParams: Map<String, Any?> = mapOf(
             FIELD_DURATION to duration?.asSeconds,
             FIELD_SELECTED_LPM to paymentSelection.defaultAnalyticsValue,
             FIELD_INTENT_TYPE to initializationMode.defaultAnalyticsValue,
             FIELD_ORDERED_LPMS to orderedLpms.joinToString(",")
+        ).plus(
+            linkMode?.let { mode ->
+                mapOf(FIELD_LINK_MODE to mode.defaultAnalyticsValue)
+            }.orEmpty()
         )
 
+        private val LinkMode.defaultAnalyticsValue: String
```

Comment:
```suggestion
        private val LinkMode.analyticsValue: String
```

Maybe I'm missing something, but `default` doesn't seem relevant here.